apps/api/src/persistence/repositories/arango-id-repository.ts

```diff
@@ -109,14 +112,21 @@ export class ArangoIdRepository implements IIdRepository {
             throw new InternalError(`Cannot reserve id: ${id} as it is already in use`);
         }
 
-        await this.arangoDatabase.update<Partial<UuidDocument>>(
-            // The sequence number is the Arango document _key
-            result.sequenceNumber,
-            {
-                usedBy: type,
-                timeUsed: new Date().toISOString(),
-            },
-            ArangoCollectionId.uuids
-        );
+        /**
+         * We are moving away from tracking historical IDs in the actual database
```

Comment:
I'd hold off on this change unless we can prove with metrics that this is a problem.

Reply:
done